src/ActionSheet/TouchableNativeFeedbackSafe.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import {
+  Platform,
+  TouchableNativeFeedback,
+  TouchableOpacity,
+  View,
+  TouchableWithoutFeedbackProps,
+} from 'react-native';
+
+// This TouchableOpacity has the same staic method of TouchableNativeFeedback
+class CustomTouchableOpacity extends TouchableOpacity {
+  static SelectableBackground = () => ({});
+  static SelectableBackgroundBorderless = () => ({});
+  static Ripple = (color: string, borderless?: boolean) => ({});
+}
+
+const TouchableComponent = Platform.select({
+  web: CustomTouchableOpacity,
+  default: Platform.Version <= 20 ? CustomTouchableOpacity : TouchableNativeFeedback,
+});
+
+type Props = TouchableWithoutFeedbackProps & {
+  pressInDelay: number;
+  background: any;
+};
+
+export default class TouchableNativeFeedbackSafe extends React.Component<Props> {
+  static SelectableBackground = TouchableComponent.SelectableBackground;
+  static SelectableBackgroundBorderless = TouchableComponent.SelectableBackgroundBorderless;
+  static Ripple = TouchableComponent.Ripple;
+
+  render() {
+    if (TouchableComponent === TouchableNativeFeedback) {
+      return (
+        <TouchableComponent {...this.props} style={{}}>
+          <View style={this.props.style}>{this.props.children}</View>
+        </TouchableComponent>
+      );
+    } else {
```

Comment:
what if instead of the else you just returned <TouchableComponent />

1 less curly to look at!

Reply:
👍